src/app/(auth)/login/page.js

```diff
@@ -52,6 +53,11 @@ export default function Login() {
             <Button type="submit" variant="contained" color="green" fullWidth sx={{ mt: 6 }}>
               Login
             </Button>
+            <Typography sx={{ mt: 2, textAlign: 'center' }}>
+              <Link href="/signup" style={{ marginLeft: '5px', fontWeight: 'bold', color: 'green' }}>
```

Comment:
Use sx={{}} for style. Why you need left margin?
Also it should be a link to sign in page at a sign up page